src/engraving/libmscore/chordlist.cpp

```diff
@@ -1432,6 +1434,799 @@ qreal ChordList::position(const QStringList& names, ChordTokenClass ctc) const
     }
 }
 
+//---------------------------------------------------------
+//   findModifierStartIndices
+//---------------------------------------------------------
+void ParsedChord::findModifierStackIndices()
```

Comment:
This method has a confusing signature. If something has the word "find" in its name, it should return some value